code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2043,3 +2043,14 @@
 			wounded_part.heal_damage(0.25, 0.25)
 		M.adjustStaminaLoss(-0.25*REM) // the more wounds, the more stamina regen
 	..()
+
+
+/datum/reagent/plaguebacteria
+	name = "Yersinia pestis"
+	description = "It is a VERY bad idea to drink this."
+	color = "#7CFC00"
+	taste_description = "death"
+	can_synth = FALSE
+
+/datum/reagent/plaguebacteria/reaction_mob(mob/living/L, method=TOUCH, reac_volume, show_message = 1, touch_protection = 0)
+	L.ForceContractDisease(new /datum/disease/plague(), FALSE, TRUE)
```

Comment:
Might want to remove all instances of the reagent from the victims system once it applies the disease. Currently we're adding 3u of the reagent every time a rat attacks someone... if a rat found someone incapacitated they could fill them up with a ridiculous amount of the reagent before they wake up to do anything about it.

Reply:
Ok i would try to do this